code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/configuration/ConfigurationExtension.kt

```diff
@@ -489,16 +489,10 @@ internal class ConfigurationExtension : Extension {
      * @param triggerEvent the [Event] to which the response is being dispatched
      */
     private fun dispatchConfigurationResponse(eventData: Map<String, Any?>, triggerEvent: Event?) {
-        val builder = Event.Builder(
+        val event = Event.Builder(
             "Configuration Response Event",
             EventType.CONFIGURATION, EventSource.RESPONSE_CONTENT
-        ).setEventData(eventData)
-
-        val event: Event = if (triggerEvent == null) {
-            builder.build()
-        } else {
-            builder.inResponseToEvent(triggerEvent).build()
```

Comment:
`dispatchConfigurationResponse(..)`  should always honor the trigger event that is passed.
I think the fix here to maintain [parity with swift](https://github.com/adobe/aepsdk-core-ios/blob/55d208283eed4f99b5fe8ed75940fcc03b0511a1/AEPCore/Sources/configuration/Configuration.swift#L229) should be to do
`dispatchConfigurationResponse(config, null)` instead of passing the trigger event [here](https://github.com/adobe/aepsdk-core-android/blob/dev-v2.0.0/code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/configuration/ConfigurationExtension.kt#L477)

Reply:
Discussed it offline, and we will make some changes in another PR.